win32/Makefile.arm

```diff
@@ -28,6 +28,7 @@ WFLAGS  = \
 	-D_CRT_SECURE_NO_DEPRECATE \
 	-D_CRT_NONSTDC_NO_DEPRECATE \
 	-DUNALIGNED_OK \
+	-DARM_NEON_HASLD4 \
 	-D_ARM_WINAPI_PARTITION_DESKTOP_SDK_AVAILABLE=1 \
 	-DARM_FEATURES \
```

Comment:
Maybe should keep these in some kind of lexical order... Usually underscore comes before anything else though...